python_examples/intervention_example/disease_testing_simulation_flow.py

```diff
@@ -111,7 +111,7 @@
         int(label[i])))
 
 plt.legend()
-plt.title("Infection curves for case isolation following testing or not")
+plt.title("Infection curves for case isolation following testing (1) or not (0)")
```

Comment:
I'm afraid I'm going to request that we make the labels neater (i.e. Testing and no testing) rather than just changing the title

Reply:
I can't suggest changes on the file as its too far back, but even just add `legend_labels = ['No Testing', 'Testing']` below L92, and then change the label in L110 to `label = legend_labels[i]`